src/ray/protobuf/node_manager.proto

```diff
@@ -94,6 +95,23 @@ message RequestWorkerLeaseReply {
   string scheduling_failure_message = 10;
 }
 
+// Request to prestart workers. At this time we don't yet know the resource, or task type.
+message PrestartWorkersRequest {
+  Language language = 1;
+  // Owner: job or root detached actor.
+  optional bytes job_id = 2;
+  optional bytes root_detached_actor_id = 3;
```

Comment:
job_id is required and root_detached_actor_id is not supported?

Reply:
yes, this root_detached_actor_id is just for future proof

Reply:
Remove it if it's not used, we can always add it later when needed.

Reply:
done, removed